docs/content/en/docs/community/contributing.md

```diff
@@ -5,80 +5,129 @@ description: >
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
 
 * A reproducible test case or series of steps
-* The version of the code being used
+* The version of our code being used
 * Any modifications you've made relevant to the bug
 * Anything unusual about your environment or deployment
 
 ## Contributing via Pull Requests
-Contributions via pull requests are much appreciated. Before sending us a
-pull request, please ensure that:
+Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
 
 1. You are working against the latest source on the *main* branch.
-2. You check existing open, and recently merged, pull requests to make sure
-someone else hasn't addressed the problem already.
-3. You open an issue to discuss any significant work - we would hate for your
-time to be wasted.
+2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
+3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
 
 To send us a pull request, please:
 
 1. Fork the repository.
-2. Modify the source; please focus on the specific change you are contributing.
-If you also reformat all the code, it will be hard for us to focus on your change.
+2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it
+   will be hard for us to focus on your change.
 3. Ensure local tests pass.
 4. Commit to your fork using clear commit messages.
-5. Send us a pull request, answering any default questions in the pull request
-interface.
-6. Pay attention to any automated CI failures reported in the pull request
-and stay involved in the conversation.
+5. Send us a pull request, answering any default questions in the pull request interface.
+6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
 
-GitHub provides additional documentation on
-[forking a repository](https://help.github.com/articles/fork-a-repo/) and
+GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
+
 ## Finding contributions to work on
-Looking at the existing issues is a great way to find something to contribute on.
-The project uses the default GitHub issue labels
-(enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any
-['help wanted'](https://github.com/aws/eks-anywhere/labels/help%20wanted)
+Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the
+default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any 'help wanted'
```

Comment:
Do we (or should we) have a "good-first-issue" tag like Kubernetes?

Reply:
I believe we do, we just haven't used it yet but I can add it here